server/src/main/java/com/flipkart/varadhi/db/ZNode.java

```diff
@@ -140,6 +160,10 @@ public static ZNode ofOrg(String orgName) {
         return new Builder().withZNodeKind(ORG).withName(orgName).build();
     }
 
+    public static ZNode ofOrgNamedFilter(String orgName, String namedFilterName) {
```

Comment:
what is namedFilterName?

namedFilter is a payload and it wont have a name. right?

Reply:
Builder().withZNodeKind(ORG).withName(orgName).withChild(ORG_FILTER.kind()).build();